modules/tensor_mechanics/include/materials/ComputeCreepPlasticityStress.h

```diff
@@ -51,6 +51,17 @@ class ComputeCreepPlasticityStress : public ComputeMultipleInelasticStressBase
   virtual void updateQpState(RankTwoTensor & elastic_strain_increment,
                              RankTwoTensor & combined_inelastic_strain_increment) override;
 
+  void computeStress(const RankTwoTensor & elastic_strain_increment, RankTwoTensor & stress);
+  void computeTangentOperators();
+  void computeInelasticStrainIncrements(Real effective_trial_stress,
+                                        const RankTwoTensor & deviatoric_trial_stress);
+  void finalizeConstitutiveModels();
+
   PowerLawCreepStressUpdate * _creep_model;
   IsotropicPlasticityStressUpdate * _plasticity_model;
+
+  Real _effective_creep_strain_increment;
+  Real _effective_plastic_strain_increment;
+
+  RankTwoTensor _inelastic_strain_increment[2];
```

Comment:
Can this be an `std::array` of `RankTwoTensor`s?

Reply:
Yes, it could.  Why `std::array`?

Reply:
Because it's a part of the standard library and has additional functionality. An `std::array` over `std::vector` because in your use case it has a fixed size. What really caught my attention here is that I had never seen (or don't remember) using brackets on a MOOSE type --I couldn't find any on RankTwoTensor.

Reply:
OK.  I'll make that change.  In this case, we don't use anything from `std::array`.  We just need two RankTwoTensors.